### ISSUE TITLE
executable permission setting preventing use by users on multi-user Linux system

### ISSUE THREAD

**Issue by paciorek** [hrue/r-inla#89]
We're installing INLA on a Linux cluster, via `install.packages("INLA",repos=c(getOption("repos"),INLA="https://inla.r-inla-download.org/R/stable"), dep=TRUE)`, which installs the binary package.

When users try to use it, they get a permission denied error for `inla.mkl.run` and other INLA executables.

The permissions for the inla and fmesher executables in `bin/linux/64bit` in the installed package are set to 744 rather than being set to 755. I see that in `r-inla/utils/R/updateBin` line 10, you change to 755, but perhaps something is causing this not to be done when building the binary package?

**Comment by hrue** [comment]
Hi Chris!

does this happens also after you instal another binary, with
inla.binary.install()
?

**Comment by paciorek** [comment]
Running `inla.binary.install()` after the initial install causes the permissions to be set to 755, as desired.

**Comment by hrue** [comment]
I do not know what is happening when R install the package. no one else has reported this issue, so maybe it is something spesific on your side, like a more restrictive 'umask' value? it is possible to check this at startup and possible fix permissions then? You think this will be a good idea?

**Comment by paciorek** [comment]
I talked with our sysadmin here and here are a few thoughts:

1) We don't think it is anything about our system specifically. As additional evidence, when I install INLA in a Rocker container, the permissions on the binaries are:
  ```
  -rwxrw-r-- 1 root staff   774240 Jan 11 20:40 fmesher
  -rwxrw-r-- 1 root staff      482 Jan 11 20:40 fmesher.run
  -rwxrw-r-- 1 root staff 55268840 Jan 11 20:40 inla.mkl
  -rwxrw-r-- 1 root staff     1638 Jan 11 20:40 inla.mkl.run
  -rwxrw-r-- 1 root staff     1638 Jan 11 20:40 inla.run
  ```
  Given the lack of executable permissions except for the user, that suggests that installation on a multi-user system wouldn't work for the various users on the system. But I agree that if INLA had problems on a multi-user system that 
  I would expect this to have been reported before!
2.) We're wondering if it would make sense to set the mode of the binaries to 755 before you create the INLA binary package. Then I would think that mode would be retained when the binary package files are placed on the system of the user. 
3.) My colleague thought it sounded 'hacky' to check/fix permissions at startup.

In any event, we manually fixed things here on our system, so this is not a concern for us. Just wanted to let you know in case it was something you wanted to look into.
